makefiles/libfaketime.mk

```
@@ -0,0 +1,50 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS         += libfaketime
+LIBFAKETIME_VERSION := 0.9.10
+DEB_LIBFAKETIME_V   ?= $(LIBFAKETIME_VERSION)-1
```

No need for the subversion here..

```suggestion
DEB_LIBFAKETIME_V   ?= $(LIBFAKETIME_VERSION)
```

---

Some patches has been applied to the final product, so adding that revision number

---

Some patches has been applied to the final product, so adding that revision number

---

Ideally, we'd want to match the version release, not make our own version; I get why it's there though...